### PR TITLE
Cayden simple edits

### PR DESCRIPTION
**Pull request by ccodel**
While the branch name is a bit of a misnomer, this PR tightens up a fair bit of prose without (hopefully) removing any ideas in the text. The changelog of edits are as follows.

Abstract:
- Reworded "With a combination of..." in second sentence.
- "Erdos-Szekeres-type problems" -> "similar problems" since the proper nouns were not introduced in the abstract.
- Simplified ending sentence.

Section 1 (Intro):
- Tightened up prose in second paragraph
  - Swapped order of ideas to emphasize that the "reduction step is not as trustworthy"
- Tightened up prose in "Verification of SAT proofs" paragraph.
  - Starting sentence shortened to "makes it trustworthy."
  - Shorten edsentence following "the reduction step has not received similar scrutiny," and making it a dependent clause.
  - Final two sentences of Maric paragraph shorter.

Section 4 (triple orientations):
- Tightened prose in first paragraph, split into two.
- Final two paragraphs placed into prose instead of an \begin{itemize} and tightened up.
- Reviewer comment about the remark of $(\mathbb{R}^2)^n$ addressed with explanatory prose.

### CODE REVIEW

File: ITP/geometry.tex [hunk 1]
```diff
@@ -49,5 +49,6 @@
     %  Indeed, assume $s$ is not empty w.r.t. $S$, there is a point $p \in S \setminus s$ that lies inside the convex hull of $s$. Because $P$ is a partition of the convex hull of $s$, point $p$ must be inside some $t \in P \subseteq T$.
     %  To see convexity, we can reason contrapositively again. If $s$ is not convex, then there is a point $p \in s$ that lies inside the convex hull of $s$, and thus lies inside a triangle $t \in P$.
 \end{proof}
-    
-The next section shows how boolean variables can be used to encode which triangles are empty w.r.t.~a pointset, which as the previous theorem shows, can be used to encode the presence or absence of $k$-holes.
+
+In the next section,
+we show how to encode which triangles (and, by the above theorem, which $k$-holes) are empty in a pointset with boolean variables.
```
Comment (JamesGallicchio) [inline ITP/geometry.tex]: "with boolean variables" is a bad syntax tree; maybe move back to beginning of sentence: "we show how to encode with boolean variables the question of which triangles ..."

File: ITP/triple-orientations.tex [hunk 2]
```diff
@@ -1,9 +1,24 @@
...
+can be described in terms of axiomatizable relationships between points and lines
+that are invariant under rotation, translation, and even small perturbations of the points.
+We can represent these relationships with boolean variables,
+thus making us agnostic to the specific coordinates of the points.
+The most widely used combinatorial abstraction used for these kinds of problems is that of \emph{triple orientations}~\cite{ emptyHexagonNumber, scheucherTwoDisjoint5holes2020},
+which are also known as \emph{signotopes}~\cite{felsnerSweepsArrangementsSignotopes2001,subercaseaux2023minimizing},
+Knuth's \emph{counterclockwise} relation~\cite{knuthAxiomsHulls1992},
+or \emph{signatures}~\cite{szekeres_peters_2006}.
+An example is illustrated in~\Cref{fig:triple-orientation}.
+Given points $p, q, r$, their \emph{triple-orientation} is defined as:
```
Comment (JamesGallicchio) [inline ITP/triple-orientations.tex:60]: I'm not sure I agree with this rewrite; the old version reads decently well to me, even if it is a bit wordy.

File: ITP/triple-orientations.tex [hunk 3]
```diff
@@ -141,14 +157,22 @@
...
-\end{enumerate}
+The previous theorem is important for two reasons.
+First, if $\sigma$ is invariant under certain point transformations (e.g., rotations, translations, etc.), then any orientation property is invariant under the same transformations.
+Identifying these transformations allows us to perform symmetry breaking to simplify the search space while preserving the property of interest (see~\Cref{sec:symmetry-breaking}).
```
Comment (JamesGallicchio) [inline ITP/triple-orientations.tex]: "This is a powerful tool for performing symmetry breaking (see~\Cref{sec:symmetry-breaking})."

File: ITP/triple-orientations.tex [hunk 4]
```diff
@@ -141,14 +157,22 @@
...
+Since $\sigma$ is invariant under translation, we can see that this assumption indeed does not impact the validity of the proof.
+
+Second,
+as introduced at the beginning of this section,
+SAT encodings for Erd\H{o}s-Szekeres-type problems use triple orientations to capture properties like convexity or emptiness,
+thus discretizing the problem.
+Because we have proved that $\pi_k(S)$ is an orientation property,
+we find that the values of $\sigma$ for all triples of points in $S$ contain enough information to determine whether $\pi_k(S)$.
+Therefore,
+we may analyze the values of $\sigma$ for the point triples in $S$, a discrete space with at most $2^{n^3}$ possibilities,
```
Comment (JamesGallicchio) [inline ITP/triple-orientations.tex]: I think I prefer the old sentences starting at (and including) "thus discretizing the problem"

**Review by JamesGallicchio — COMMENTED** [review]
good changes, gets rid of some of our unhelpful blabbering

File: ITP/geometry.tex (hunk 1 above)
Comment (ccodel) [reply ITP/geometry.tex]: Good catch. Will fix.

File: ITP/triple-orientations.tex (hunk 2 above)
Comment (ccodel) [reply ITP/triple-orientations.tex:60]: Okay, I'll rewrite this to be closer to the original.

File: ITP/triple-orientations.tex (hunk 3 above)
Comment (ccodel) [reply ITP/triple-orientations.tex]: Incorporated.

File: ITP/triple-orientations.tex (hunk 4 above)
Comment (ccodel) [reply ITP/triple-orientations.tex]: I restored the old text, except for a phrase about "grappling with a continuous search space," as this was a request by reviewer 3.

**Comment by ccodel** [comment]
Changes made. What do you think?

**Comment by bsubercaseaux** [comment]
Looks good, I agree with James' comments and they all seem addressed now.
